README.md

```diff
@@ -72,6 +72,14 @@ aws.bedrock.model.llama2.maxTokens=200
 aws.bedrock.model.llama2.temperature=0.5
 ```
 
+#### Titan model parameters
+
+```properties
+aws.bedrock.model.id=amazon.titan-text-express-v1
```

Comment:
This is the right level,  then what is "aws.bedrock.model.titan.id" for .. it gets a little confusing. Let's chat about what the props should look like for each model. I would prefer not to activate more than one model at a time unless you convince me it is a good idea.

Reply:
This is a mistake.  Should be aws.bedrock.model.titan.id. I'll push an update to the docs.  Behind the scenes, this is the case.

Reply:
I think we shouldn't limit the number of services activated.  Consumers could specify @Primary and @Qualifier annotations to override auto-config.